### ISSUE TITLE
Wine 에서 실행 시 오류

### ISSUE THREAD

**Issue by yeodakyum** [KurtBestor/Hitomi-Downloader#3483]
와인에서 2.8버전부터 실행시 "internal error cannot create temporary directory" 가 발생합니다.
py installer 3.6버전 이상에서 발생하는 문제 (https://github.com/pyinstaller/pyinstaller/issues/4628)같은데, 혹시 3.5버전으로 패키징 해 주실 수 있으신가요?

**Comment by KurtBestor** [comment]
관리자 권한으로 실행해도 안 되나요?

**Comment by yeodakyum** [comment]
네, 찾아 보니까 pyinstaller 3.6 에서 --onefile 옵션을 사용해서 패키징 하면 발생하는거 같네요.

**Comment by KurtBestor** [comment]
Wine 하나 때문에 다운그레이드 하고 싶진 않네요

Wine 에서 고쳐야할 문제 같아요

**Comment by yeodakyum** [comment]
제가 테스트 코드로 해보니까 --onefile 옵션 없이 하면 3.6에서도 정상 작동하는거 같은데, 혹시 그 옵션 없이 해 주실 수 있나요..? wine 내부 버그 인거 같아서 해결이 어렵습니다

**Comment by KurtBestor** [comment]
```hitomi_downloader_GUI.exe --safemode``` 로 실행해보세요

**Comment by idisaj93** [comment]
혹시 해결하셨나요?

**Comment by idisaj93** [comment]
> `hitomi_downloader_GUI.exe --safemode` 로 실행해보세요

안녕하세요 최신버전에서 고쳐졌다는듯한데 pyinstaller를 최신버전으로 업데이트해주실수 없을까요
